rectools/models/sasrec.py

```diff
@@ -1293,7 +1300,69 @@ def torch_model(self) -> TransformerBasedSessionEncoder:
 
 
 class SASRecModel(TransformerModelBase):
-    """TODO"""
+    """
+    SASRec model for i2i and u2i recommendations.
+
+    n_blocks: int, default 1
+        Number of transformer blocks.
+    n_heads: int, default 1
+        Number of attention heads.
+    n_factors: int, default 128
+        Latent embeddings size.
+    use_pos_emb: bool, default ``True``
+        If ``True``, adds learnable positional encoding to session item embeddings.
+    use_causal_attn: bool, default ``True``
+        If ``True``, uses causal mask as attn_mask in Multi-head Attention.
+    use_key_padding_mask: bool, default ``False``
+        If ``True``, uses key_padding_mask in Multi-head Attention.
+    dropout_rate: float, default 0.2
+        Probability of a hidden unit to be zeroed.
+    session_max_len: int, default 32
+        Maximum length of user sequence.
+    train_min_user_interaction: int, default 2
+        Minimum number of interactions user should have to be used for training. Should be greater than 1.
+    dataloader_num_workers: int, default 0
+        Number of loader worker processes.
+    batch_size: int, default 128
+        How many samples per batch to load.
+    loss: str, default "softmax"
+        Loss function.
+    n_negatives: int, default 1
+        Number of negatives for BCE and gBCE losses.
+    gbce_t: float, default 0.2
+        Calibration parameter for gBCE loss.
+    lr: float, default 0.01
+        Learning rate.
+    epochs: int, default 3
+        Number of training epochs.
+    verbose: int, default 0
+        Verbosity level.
+    deterministic: bool, default ``False``
+        If ``True``, sets deterministic algorithms for PyTorch operations.
+        Use `pytorch_lightning.seed_everything` together with this parameter to fix the random state.
+    recommend_device: str, default "auto"
+        Device for recommend. Used at predict_step of lightning module.
```

Comment:
Please add this to all 3 arguments with the same logic:
```
If you want to change this parameter after model is initialized,
you can manually assign new value to model `recommend_<param>` attribute.
```

Reply:
fixed